moped-api/users/validation.py

```diff
@@ -27,7 +27,7 @@
         "nullable": False,
         "required": True,
         "empty": False,
-        "regex": "^[a-zA-Z0-9 \-\.\']+$",
+        "regex": "^[a-zA-Z0-9 \-\.']+$",
```

Comment:
Black formatter removed these escapes, but they still work when I just tested on regex101.com so I guess they weren't needed in the first place. This is reminding me about [this discussion](https://github.com/cityofaustin/atd-moped/pull/740#issuecomment-1202769454) about whether we really need to validate first and last name fields (and some others)...